src/tests/components/AccountSettings/ClientProfilePage.test.tsx

```diff
@@ -0,0 +1,47 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import '@testing-library/jest-dom/extend-expect';
+
+import {
+  fireEvent, render, waitFor,
+} from '@testing-library/react';
+import fetchMock from 'jest-fetch-mock';
+import React from 'react';
+import { Provider } from 'react-alert';
+import AlertTemplate from 'react-alert-template-basic';
+
+import { ClientProfilePage } from '../../../components/AccountSettings/ClientProfilePage';
+
+fetchMock.enableMocks();
+
+describe('ClientProfilePage', () => {
+  beforeEach(() => {
+    fetchMock.resetMocks();
+  });
+
+  describe('Upload Profile Photo', () => {
+    test('should call backend to upload photo', async () => {
+      // Arrange
+      fetchMock.mockResponseOnce(JSON.stringify({ status: 'SUCCESS' }));
+
+      const alertShowFn = jest.fn();
+      const file = new File(['profile photo'], 'photo.png', { type: 'image/png' });
```

Comment:
feel free to use an actual image from the src/static/img folder here just so you are simulating a real image